docs/releases.md

```diff
@@ -1,39 +1,157 @@
-the five weeks is up, a release candidate is tagged and only important bugfixes
-will be allowed up to release day.
+## Table of Contents
 
-## Release Candidate Checklist
-- [ ] CHANGELOG.md has been updated
-  - use `./bin/mkreleaselog` to generate a nice starter list
+- [Release Philosophy](#release-philosophy)
+- [Release Flow](#release-flow)
+- [Performing a Release](#performing-a-release)
+- [Release Version Numbers (aka semver)](#release-version-numbers-aka-semver)
+
+## Release Philosophy
+
+`go-ipfs` aims to have release every six weeks, two releases per quarter. During these 6 week releases, we go through 4 different stages of Release Candidates (RC) that gives us the opportunity to test the new version against our test envinronments, production envinronments, IPFS apps (e.g. Desktop and WebUI) and with our close partners that have IPFS running in Production, by leveraging their own test infrastructure.
+
+We might expand the six week release schedule in case of:
+- No new updates to be added
+- In case of a large community event that takes the core team availability away (e.g. IPFS Conf, Dev Meetings, IPFS Camp, etc)
+
+## Release Flow
+
+`go-ipfs` releases come in 4 stages:
+
+- **Release Stage 1 - Internal testing** - Test the Release against our testing infrastructure, including interoperability, integration, test lab, multiple runtimes and the apps we've built (WebUI, Desktop, NPM on IPFS, HTTP Client Libraries). The intent is to make this stage fully automated (and somewhat is already), until then, we manually check a list and ensure that all tests have been run
+- **Release Stage 2 - Invite IPFS partners to test** - Reach out to our partners (i.e. projects that have volunteered to support `go-ipfs` by using their own test infrastructure and tell us the results)
+- **Release Stage 3 - Announce to the broader community** - Communicate to the community that a new Release Candidate is ready and that everyone is welcome to test it with us
+- **Release Stage 4 - Complete the Release** - Finalize the release, start the next release.
+
+The Release Stages are not linked to Release Candidate numbers, in fact, there can be multiple release candidate per stages as we catch bugs and improve the release itself.
+
+<p align="center">
+  <a href="https://ipfs.io">
+    <img src="https://gateway.ipfs.io/ipfs/QmaFtLxoCAm5vFQ9AftKkhJwSAdDdF1jzV9DfzW6gbXqFL/Paper.Sketches.23.png" width="450" />
+  </a>
+</p>
+
+## Performing a Release
+
+The first step is for the `Lead Maintainer` for `go-ipfs` to open an issue with Title `go-ipfs <version> Release` and a c&p of the following template:
+
+```
+> <short tl;dr; of the release>
+
+# 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+# 🔦 Highlights
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+# 🏗 API Changes
+
+<List of API changes, if any>
```

Comment:
The `🔦 Highlights` and `🏗 API Changes` could potentially be replaced by the Changelog. That said, there is still value on highlighting some of the changes, the ones that we want users to know about first (or the ones that we want to ensure that users see, in case they don't have time to read the entire changelog).

Reply:
Or, more generally, breaking changes/notices.